composer.lock

```diff
@@ -1129,22 +1175,6 @@
             "require": {
                 "npm-asset/ev-emitter": ">=1.0.0,<2.0.0"
             },
-            "require-dev": {
```

Comment:
Why do we remove all `require-dev` ?

Reply:
Composer magic. I have no idea.

Reply:
strange ..

Reply:
It's on NPM virtual packages so I'm not too worried about it.